packages/compass-databases-navigation/src/collection-item.tsx

```diff
@@ -130,22 +132,29 @@ export const CollectionItem: React.FunctionComponent<
       isActive={isActive}
       isTabbable={isTabbable}
       onDefaultAction={onDefaultAction}
-      className={collectionItem}
       style={style}
       {...hoverProps}
     >
-      <CollectionIcon type={type} />
-      <ItemLabel className={collectionItemLabel} title={name}>
-        {name}
-      </ItemLabel>
-      <ItemActionControls<Actions>
-        className={collectionActions}
-        onAction={onAction}
-        data-testid="sidebar-collection-item-actions"
-        iconSize="small"
-        isVisible={isActive || isHovered}
-        actions={actions}
-      ></ItemActionControls>
+      <ItemWrapper numIcons={actions.length}>
```

Comment:
I think relying on actions length here gives weird results sometimes, for example collections will have from 2 to 4 actions available, always would show one icon here as this behavior is based on the `collapseToMenuThreshold` prop of the `ItemActionControls`, not on the actual value of the items

```suggestion
      <ItemWrapper numIcons={1}>
```